install.ps1

```diff
@@ -21,16 +21,20 @@ $VENV = "$HOME\venvs\$COURSE"
 function is-m269-folder {
     param($path)
     if (-not (Test-Path $path -PathType Container)) {
+        $folder = $path
         $msg="doesn't exist or isn't a folder"
     }
-    elseif (-not ($path -match "[Mm]269-23[Jj]$")) {
-        $msg="must be named m269-23j or M269-23J"
-    }
-    else
-    {
-        return
+    else {
+        $folder = Convert-Path $path
+        if (-not ($folder -match "[Mm]269-23[Jj]$")) {
```

Comment:
When any relative paths are expanded, it seems there is a trailing `\` backslash which needs to be added to the pattern here (in the form of an escaped backslash `\\`). I believe this is because of matching the end of the line i.e the `$` 

In the Unix script we use the `basename` command which I am assuming strips any trailing slashes so does not suffer the same issue.

Reply:
So the pattern should be `...[Jj][\\]$` to allow for a potential trailing backslash? I actually don't know if it should be  instead `[Jj]\\?` or `[Jj](\\)?`

Reply:
@densnow I think it's best you correct the pattern yourself, test it and then merge the PR, instead of us bouncing messages here and me doing commits that may not work.

Reply:
@mwermelinger Yes no worries, I will make the changes. 

I was trying to think if there is a case in the Windows script where an argument (that is the path) will not be passed to the `Convert-Path` command and came to the conclusion that it would not. Although having said that I guess going with 0 or 1 pattern won't do any harm and is probably the safest way, so I will go with that.